server/src/Metrics.hs

```diff
@@ -61,16 +54,19 @@ createAndRegisterIcepeakMetrics = IcepeakMetrics
                               "Total number of items removed from the queue.."))
   <*> register (histogram (Info "icepeak_sync_duration" "Duration of a Sync command.")
                           syncBuckets)
-  <*> register (histogram (Info "icepeak_http_req_handling_duration"
-                                "Duration of the handling of an HTTP request.")
-                          defaultBuckets)
   where
-    requestCounter = counter (Info "icepeak_http_requests"
-                                   "Total number of HTTP requests since starting Icepeak.")
-    syncBuckets = exponentialBuckets 0.001 2 12
+    requestHistogram = histogram (Info "icepeak_http_requests"
```

Comment:
We should choose a different name following the naming convention for prometheus metrics: https://prometheus.io/docs/practices/naming/

Among the names they give as an example, they give `http_request_duration_seconds` which I think is applicable here. Note that an application prefix in the name is explicitly not included since HTTP requests are a generic domain.
We can still distinguish them because prometheus will remember which service it scraped a metric from.